assets/charts/components/aws-ebs-csi-driver/values.yaml

```diff
@@ -84,6 +84,3 @@ serviceAccount:
     annotations: {}
   snapshot:
     annotations: {}
-
-# False if you want to use a different storage option, e.g. OpenEBS
-enableDefaultStorageClass: true
```

Comment:
This seems like our change. Should this be moved to component values?

Reply:
Is it our change ?

I didn't see the `XXX Lokomotive specific change` comment.

Reply:
Then I'd move it to component code to avoid confusion.

Reply:
ping @ipochi

Reply:
@invidian This is already part of the templated `values.yaml`

https://github.com/kinvolk/lokomotive/blob/4b4e249506b2ca85a7c897ba64c4ec0bb8116a62/pkg/components/aws-ebs-csi-driver/component.go#L32

Reply:
I thought we just hardcode the value here. Sorry for the confusion. In this case, this should stay here as a documentation and a default value for value used by the Helm chart. It shouldn't be removed.

Reply:
@invidian 

I haven't removed it manually. Its not part of the helm chart.
I suppose it was added by us in the previous version of the chart, but its not present in 0.6.0 version of the upstream chart.

Reply:
Yes, it seems like our addition used here:
```
$ git grep enableDefaultStorageClass
assets/charts/components/aws-ebs-csi-driver/templates/storageclass.yaml:{{- if .Values.enableDefaultStorageClass }}
```

Perhaps we can add `XXX Lokomotive specific change` then.

Actually, it seems entire storage class manifest is our addition. We should upstream it.

Reply:
Opened an issue upstream: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/605.

Reply:
And created a PR: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/606

Reply:
please re-review.